src/features/donations/components/Donation/ShareDonation/ShareDonation.js

```diff
@@ -0,0 +1,72 @@
+import {
+  FacebookShareButton,
+  WhatsappShareButton,
+  TelegramShareButton,
+  TwitterShareButton,
+  RedditShareButton,
+  EmailShareButton,
+} from 'react-share';
+
+import { ReactComponent as WhatsappIcon } from 'assets/whatsapp_icon.svg';
+import { ReactComponent as FacebookIcon } from 'assets/facebook_icon.svg';
+import { ReactComponent as TelegramIcon } from 'assets/telegram_icon.svg';
+import { ReactComponent as TwitterIcon } from 'assets/twitter_icon.svg';
+import { ReactComponent as RedditIcon } from 'assets/reddit_icon.svg';
+import { ReactComponent as EmailIcon } from 'assets/email_icon.svg';
+import ShareButton from './ShareButton';
+
+const ShareDonation = () => {
+  const currentURL = window.location.href;
+  const description =
+    "Hola! hecha un vistazo de lo que estan donando en 'Che, ¿hoy donamos?'";
+
+  return (
+    <div className="flex mt-2">
+      <ShareButton className="share-button__whatsapp">
+        <WhatsappShareButton title={description} url={currentURL}>
+          <WhatsappIcon />
+        </WhatsappShareButton>
+      </ShareButton>
+      <ShareButton className="share-button__telegram">
+        <TelegramShareButton title={description} url={currentURL}>
+          <TelegramIcon />
+        </TelegramShareButton>
+      </ShareButton>
+      <ShareButton className="share-button__twitter">
+        <TwitterShareButton title={description} url={currentURL}>
+          <TwitterIcon />
+        </TwitterShareButton>
+      </ShareButton>
+      <ShareButton className="share-button__reddit">

```

Comment:
`className` looks weird, maybe we could add the name of the social media as a prop called `type`, and then in the component, we could try ```styles[`share-button__${type}`]```, wdyt?